src/plugins/intel_cpu/src/cpu_memory.cpp

```diff
@@ -485,9 +488,6 @@ MemoryBlockPtr StaticMemory::getMemoryBlock() const {
 
 // oneDNN specifics for backward compatibility
 dnnl::memory StaticMemory::getPrimitive() const {
-    if (!m_prim) {
-        OPENVINO_THROW("Couldn't create dnnl::memory object: ", dnnlErrorCtx);
-    }
```

Comment:
At this point we don't know whether the primitve is null because of the "empty" descriptor or because of the oneDNN internal error. Could you please bring the check back, but enhance it with testing the type of the memory descriptor to distinguish the valid case of an empty desc and an internal dnnl memory error.
Apparently the best solution would be to remove any dnnl specifics from the plugin memory object, but as long as we keep it in this entity, we have to properly process errors.

Reply:
The problem is this is how empty entities are implemented in onednn
С++ oneDNN object is just a shared_ptr wrapper around C object.
And C++ object is empty if this shared_ptr is nullptr;
So, this is impossible to distinguish valid and invalid states.
There is basically no invalid state.

Reply:
We can check the plugin's memory descriptor:
https://github.com/openvinotoolkit/openvino/blob/58b02fb88fa660468d281e751f117c51cb22932c/src/plugins/intel_cpu/src/memory_desc/cpu_memory_desc.h#L38